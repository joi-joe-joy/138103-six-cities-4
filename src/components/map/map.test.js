import React from "react";
import renderer from "react-test-renderer";
import Map from "./map.jsx";

jest.mock(`leaflet`);

const coordinations = [
  [52.3909553943508, 4.929309666406198],
  [52.3809553943508, 4.939309666406198]
];

describe(`Render Map`, () => {
  it(`Renderer Map correctly`, () => {
    const tree = renderer.create(
        <Map
          offersCords={coordinations}
        />, {
          createNodeMock: () => {}
        }
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Renderer Map correctly with current`, () => {
    const tree = renderer.create(
        <Map
          currentCords={coordinations[0]}
          offersCords={coordinations}
        />, {
          createNodeMock: () => {}
        }
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
