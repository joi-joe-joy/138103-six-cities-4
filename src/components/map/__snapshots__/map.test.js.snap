// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Map Renderer Map correctly 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;

exports[`Render Map Renderer Map correctly with current 1`] = `
<div
  id="map"
  style={
    Object {
      "height": "100%",
    }
  }
/>
`;
